### DISCUSSION TITLE
any css parser built on unified?

### DISCUSSION THREAD

**Discussion by shtse8** [rehypejs/rehype#106] [General] [answered]
it's cool that rehype handled all html stuff for us.
but why  there is no unified css parser?

**Comment by ChristianMurphy** [comment, marked as answer]
There is https://github.com/shawnbot/sast

**Reply by shtse8** [reply]
It seems the project is not active anymore.
Also, it would be better we can make use of `.use()` chain.

**Comment by wooorm** [comment]
It should be possible to make something on top of [postcss](https://postcss.org/api/), [swc](https://rustdoc.swc.rs/swc_css_parser/index.html), or [parcel/css](https://docs.rs/parcel_css/1.0.0-alpha.29/parcel_css/stylesheet/struct.StyleSheet.html), too, mapping those ASTs to unist.
